script/upgrades/MU01-mcmint/governance/MU01-Baklava-CGP.sol

```diff
@@ -17,26 +17,210 @@ import { Contracts } from "script/utils/Contracts.sol";
+import { TradingLimits } from "mento-core/contracts/common/TradingLimits.sol";
+import { SortedOracles } from "mento-core/contracts/SortedOracles.sol";
 
 /**
  forge script {file} --rpc-url $BAKLAVA_RPC_URL 
                      --broadcast --legacy 
  * @dev depends on: ../deploy/*.sol
  */
 contract MU01_BaklavaCGP is GovernanceScript {
+  using TradingLimits for TradingLimits.Config;
+
   ICeloGovernance.Transaction[] private transactions;
 
+  PoolConfiguration private cUSDCeloConfig;
+  PoolConfiguration private cEURCeloConfig;
+  PoolConfiguration private cBRLCeloConfig;
+  PoolConfiguration private cUSDUSDCConfig;
+
+  address private cUSD;
+  address private cEUR;
+  address private cBRL;
+  address private celo;
+  address private USDCet;
+
+  address payable private breakerBoxProxyAddress;
+  address private cUSDUSCDRateFeedId = address(uint256(keccak256(abi.encodePacked("USDCUSD"))));
+
+  // Helper mapping to store the exchange IDs for the reference rate feeds
+  mapping(address => bytes32) private referenceRateFeedIDToExchangeId;
+
   function prepare() public {
+    loadDeployedContracts();
+    setAddresses();
+    setUpPoolConfigs();
+  }
+
+  /**
+   * @dev Loads the deployed contracts from the previous deployment step
+   */
+  function loadDeployedContracts() public {
     contracts.load("MU01-00-Create-Proxies", "1674224277");
-    contracts.load("MU01-01-Create-Nonupgradeable-Contracts", "1674224321");
-    contracts.load("MU01-02-Create-Implementations", "1674225880");
+    contracts.load("MU01-01-Create-Nonupgradeable-Contracts", "1676477381");
+    contracts.load("MU01-02-Create-Implementations", "1676504104");
+    contracts.load("MU01-04-Create-MockUSDCet", "1676392537");
+  }
+
+  /**
+   * @dev Sets the addresses of the various contracts needed for the proposal.
+   */
+  function setAddresses() public {
+    cUSD = contracts.celoRegistry("StableToken");
+    cEUR = contracts.celoRegistry("StableTokenEUR");
+    cBRL = contracts.celoRegistry("StableTokenBRL");
+    celo = contracts.celoRegistry("GoldToken");
+    USDCet = contracts.deployed("MockERC20");
+
+    breakerBoxProxyAddress = contracts.deployed("BreakerBoxProxy");
+  }
+
+  /**
+   * @dev Sets the various values needed for the configuration of the new pools.
+   *      This function is called by the governance script runner.
+   */
+  function setUpPoolConfigs() public {
+    // TODO: -> Finish adding trading limit configuration values to
+    //          the pool configs below [Tobi]
+
+    // Create pool configuration for cUSD/CELO pool
+    cUSDCeloConfig = PoolConfiguration({
+      asset0: cUSD,
+      asset1: celo,
+      isConstantSum: false,
+      spread: FixidityLib.newFixedFraction(25, 10000), // 0.0025
+      referenceRateResetFrequency: 60 * 5,
+      minimumReports: 5,
+      stablePoolResetSize: 72e23, // 7200000
+      isMedianDeltaBreakerEnabled: true,
+      medianDeltaBreakerThreshold: 3e16, // 0.03
```

Comment:
What is the preferred representation of decimals 
`FixidityLib.newFixedFraction(25, 10000)` versus `25e14`?

Reply:
Similar question regarding time convention: unit free in seconds (`60 * 5 `) versus with time unit (`5 minutes`)

Reply:
In both cases, I prefer the latter. For me, it makes them easier to read.